### PR TITLE
Reassign workbasket

### PR DESCRIPTION
**Pull request by KyleMacPherson**
https://uktrade.atlassian.net/jira/software/projects/TARIFFS/boards/127?selectedIssue=TARIFFS-649
https://uktrade.atlassian.net/jira/software/projects/TARIFFS/boards/127?selectedIssue=TARIFFS-650
https://uktrade.atlassian.net/jira/software/projects/TARIFFS/boards/127?selectedIssue=TARIFFS-651

### CODE REVIEW

File: app/views/workbaskets/reassigns/new.html.slim [hunk 1]
```diff
@@ -0,0 +1,38 @@
+script
+  == "window.save_url = '#{cross_check_url(@workbasket.id)}';"
+  == "window.__cross_check_json = #{(params[:cross_check] || {}).to_json};"
```
Comment (steveholmes) [inline app/views/workbaskets/reassigns/new.html.slim]: Are these necessary?

File: config/routes.rb [hunk 2]
```diff
@@ -180,6 +180,12 @@
       end
     end
 
+    resources :reassigns do
```
Comment (steveholmes) [inline config/routes.rb]: We only use 'new' and 'create', so should limit that here with "only: [:new, :create]" ?

File: config/routes.rb [hunk 3]
```diff
@@ -180,6 +180,12 @@
...
+      member do
+        get :submitted_for_cross_check
```
Comment (steveholmes) [inline config/routes.rb]: Is this used/necessary?

File: app/views/workbaskets/reassigns/new.html.slim [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+        - if @errors.present?
+          span.error-message
+            - @errors.each do |error|
+              = error
+      .bootstrap-row
+        .controls
+          - @users.each do |user|
+            .cross-check-decision
+              .multiple-choice
+                input type='radio' id="radioID" class="radio-inline-group js-cross-check-decision" name="user_id" value="#{user.id}" required="true"
```
Comment (steveholmes) [inline app/views/workbaskets/reassigns/new.html.slim]: When clicking on the user's name in the form, it always selects the top user and not the user I clicked on.

**Comment by KyleMacPherson** [comment]
Thanks for the review @steveholmes, addressed your comments!

**Pull request merged**